docs: automate role assignments and improve troubleshooting guide (#754)
Azure/aad-pod-identity: docs/readmes/README.troubleshooting.md

-A aad-metadata -j RETURN
```

### Run a pod to validate your identity setup

You could run the following commands to validate your identity setup (assuming you have the proper `AzureIdentity` and `AzureIdentityBinding` deployed):

```bash

> kubectl run azure-cli -it --image=mcr.microsoft.com/azure-cli --labels=aadpodidbinding=<your identity binding> /bin/bash

Reviewer: ```suggestion
kubectl run azure-cli -it --image=mcr.microsoft.com/azure-cli --labels=aadpodidbinding=<selector defined in AzureIdentityBinding> /bin/bash
```
